### PR TITLE
New Mission: Fillum Resources Claim

### PR DESCRIPTION
**Pull request by OcelotWalrus**
Adds a a new mission, and do some stuff to the missions panel. (part of closed #960)

### CODE REVIEW

File: mods/hv/maps/magma-breach-claim/magma-breach-claim.lua [hunk 1]
```diff
@@ -0,0 +1,43 @@
+--[[
+   Copyright 2021-2023 The OpenHV Developers (see AUTHORS)
```
Comment (Mailaender) [inline mods/hv/maps/magma-breach-claim/magma-breach-claim.lua:2]:
```suggestion
   Copyright 2024 The OpenHV Developers (see AUTHORS)
```

File: mods/hv/missions.yaml [hunk 2]
```diff
@@ -1,3 +1,13 @@
 Introduction:
 	itrium
 	ice-river
+
+Yuruki Campaign:
+	magma-breach-claim
+
+Synapol Campaign:
+	crater-defense
+	harbor-defense
```
Comment (Mailaender) [inline mods/hv/missions.yaml:10]: These are not added in this pull request.

File: mods/hv/missions.yaml [hunk 3]
```diff
@@ -1,3 +1,13 @@
...
+
+Misc:
+	firewatch
```
Comment (Mailaender) [inline mods/hv/missions.yaml:13]: Please remove that. `Misc` is not a useful category. Then default to none.

File: mods/hv/maps/magma-breach-claim/map.yaml [hunk 4]
```diff
@@ -0,0 +1,1163 @@
...
+		Location: 16,33
+	Actor24: goldball
+		Owner: Neutral
+		Location: 18,38
+	Actor25: goldball
+		Owner: Neutral
+		Location: 24,41
+	Actor26: goldball
+		Owner: Neutral
+		Location: 24,38
```
[the remark this answers is not in the document]
Comment (OcelotWalrus) [reply mods/hv/maps/magma-breach-claim/map.yaml:139]: Oopsie.

File: mods/hv/missions.yaml (hunk 2 above)
Comment (OcelotWalrus) [reply mods/hv/missions.yaml:10]: Yeah, missed that.

**Pull request merged**